src/ippair-bit.c

```diff
@@ -38,7 +38,7 @@
 #include "util-unittest.h"
 #include "ippair-storage.h"
 
-static int ippair_bit_id = -1;                /**< IPPair storage id for bits */
+static IPPairStorageId ippair_bit_id = { .id = -1 }; /**< IPPair storage id for bits */
```

Comment:
Preexisting but suggest remaining this var to add a prefix, e.g. `g_ippair_bit_id` or perhaps `g_ippair_bit_storage_id`